_episodes/20-crop-raster-data.md

```diff
@@ -212,4 +323,109 @@ raster_clip_wells.plot.imshow(ax=ax2)
 > > {: .language-python}
 > > <img src="../fig/20-crop-raster-dike-solution-07.png" title="Raster croped by buffer around dikes" width="512" style="display: block; margin: auto;" />
 > {: .solution}
-{: .challenge}
\ No newline at end of file
+{: .challenge}
+
+## Crop raster data using another raster data
+
+Imagine that we have two raster data, let's say the `scene` and `crop_fields` in
+different coordinate systems. Our goal is to crop the `scene` image using the
+`crop_fields` image.
+
+> ## Using `crop_fields` raster image
+>
+> For this section, we will use the `crop_fields.tif` image that was produced in
+> the exercise "**Select the raster data within crop fields**".
+{: .callout}
+
+We read in both images and check their coordinate system:
+~~~
+# Read crop_fields
+crop_fields = rioxarray.open_rasterio("crop_fields.tif")
+CRS(crop_fields.rio.crs)
+~~~
+{: .language-python}
+
+~~~
+<Derived Projected CRS: EPSG:28992>
+Name: Amersfoort / RD New
+Axis Info [cartesian]:
+- [east]: Easting (metre)
+- [north]: Northing (metre)
+Area of Use:
+- undefined
+Coordinate Operation:
+- name: unnamed
+- method: Oblique Stereographic
+Datum: Amersfoort
+- Ellipsoid: Bessel 1841
+- Prime Meridian: Greenwich
+~~~
+{: .output}
+
+~~~
+# Read scene
+items = pystac.ItemCollection.from_file("search.json")
+scene = rioxarray.open_rasterio(items[1].assets["visual"].href)
+CRS(scene.rio.crs)
+~~~
+{: .language-python}
+
+~~~
+<Derived Projected CRS: EPSG:32631>
+Name: WGS 84 / UTM zone 31N
+Axis Info [cartesian]:
+- [east]: Easting (metre)
+- [north]: Northing (metre)
+Area of Use:
+- undefined
+Coordinate Operation:
+- name: UTM zone 31N
+- method: Transverse Mercator
+Datum: World Geodetic System 1984
+- Ellipsoid: WGS 84
+- Prime Meridian: Greenwich
+~~~
+{: .output}
+
+We can see that these images are in different coordinate systems. Now, we can
+use `rioxarray.reproject_match()` function to crop `scene` image. This might
+take a few minutes, because the `scene` image is large.
```

Comment:
This should already be loaded from the beginning of the episode as `raster` variable, right? Maybe we could use a more descriptive name (something like `visual`, or `true_color_image` ) and avoid re-downloading it at this stage?

Reply:
thanks, I replaced `raster` with `true_color_image` and also removed the re-loading part.